PilotDataPlatform/search
# PILOT-1538: Update elasticsearch term key for item parent path

## Summary

- For fetching item activity logs, if parent_path was used, elasticsearch was querying `'parent_path'`, which did not exist as a doc field. The correct field is `'item_parent_path'`, which the service now uses.

## JIRA Issues

https://indocconsortium.atlassian.net/browse/PILOT-1538

## Type of Change

Please delete options that are not relevant.


## Testing

Are there any new or updated tests to validate the changes?


## Test Directions

- The number of upload and download for a particular project  "Charts" section should now be accurate.
